.github/workflows/test-backends.yml

```diff
@@ -3,7 +3,11 @@ name: Backend Tests
 on:
   workflow_dispatch:
   push:
-    branches: [main]
+    branches: [main, backend_tests_2]
```

Comment:
I might be misunderstanding the intent here, so please forgive me if i am, but the change you're making here has says "automatically run this workflow on pushes to branches named "backend_tests_2"  which seems wrong even for local testing.

You should be able to dispatch this workflow manually from any branch without a change here, right?